.github/workflows/python-bot-build.yml

```diff
@@ -0,0 +1,37 @@
+name: Python Bot Build
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies for sdk
+        run: npm run python:install
+      
+      - name: Build python sdk
+        run: npm run python:build
+      
+      - name: Link python sdk
+        run: npm run python:publish:local
+
+      - name: Install bot dependencies
+        working-directory: ./starter-project/py
+        run: npm run install
+      
+      - name: Install local python sdk on bot
```

Comment:
Not sure what else we can do after this step. Do you think it is worth having a docker build in this workflow to verify an image is correctly made?